### ISSUE TITLE
Allow lookups in ClojureDocs for ClojureScript projects

### ISSUE THREAD

**Issue by victorb** [liquidz/vim-iced#203]
I have the symbol `defn` under my cursor and press `<Leader>hc` to trigger `IcedClojureDocsOpen` but instead of seeing the docs, I'm getting the following error:

```
Tried searching for "cljs.core/defn", but ClojureDocs does not have docs for ClojureScript.
```

Which makes sense and it's true. But I wish there was a way to read the docs for the Clojure function `defn` even if I'm in a ClojureScript context/session, maybe via additional parameter to `IcedClojureDocsOpen` that I could configure myself. Or maybe I'm missing something else obvious to get things working as I want?

Otherwise, thanks for a awesome project, currently trying it out as a replacement for vim-fireplace and only scratched the surface of vim-iced but seems lovely so far.

**Comment by liquidz** [comment]
@victorb Thanks for your reporting!
Currently there is no way to use IcedClojureDocsOpen in cljs files.
So I added `g:iced#clojuredocs#use_clj_docs_on_cljs` option (default `v:false`) in `dev` branch for trial.

Could you try `dev` branch and enable this option?

**Comment by liquidz** [comment]
Closed by #204
